network-config-analyzer/nca.py

```diff
@@ -79,24 +79,24 @@ def run_args(args):
     output_format = args.output_format or 'txt'
     cl_execute = CLExecute(base_ns_list, base_pod_list, ns_list, pod_list, output_format, args.file_out, args.pr_url)
     if args.equiv:
-        return cl_execute.equivalence(args.equiv, base_np_list)
+        return cl_execute.execute_pair_configs_query('twoWayContainment', args.equiv, base_np_list)
 
     if args.interferes:
-        return cl_execute.interferes(args.interferes, base_np_list)
+        return cl_execute.execute_pair_configs_query('interferes', args.interferes, base_np_list)
 
     if args.forbids:
-        return cl_execute.forbids(args.forbids, base_np_list)
+        return cl_execute.execute_pair_configs_query('forbids', base_np_list, args.forbids)
 
     if args.permits:
-        return cl_execute.permits(args.permits, base_np_list)
+        return cl_execute.execute_pair_configs_query('permits', base_np_list, args.permits)
 
     if args.connectivity:
-        return cl_execute.connectivity_map(args.connectivity)
+        return cl_execute.execute_single_config_query('connectivityMap', args.connectivity)
 
     if args.semantic_diff:
-        return cl_execute.semantic_diff(args.semantic_diff, base_np_list)
+        return cl_execute.execute_pair_configs_query('semanticDiff', base_np_list, args.semantic_diff)
 
-    return cl_execute.sanity(args.sanity or 'k8s')
+    return cl_execute.execute_single_config_query('sanity', args.sanity or 'k8s')
 
 
 def nca_main(argv=None):
```

Comment:
the base_np_list and args.<queries> were switched un tis file for some queries in order to keep the "source" behavior
since I send the network_configs as an array in the clexecute file and take the first element of the array as the base_config.
The interferes was changed as discussed in [https://github.com/IBM/network-config-analyzer/issues/104]( ), the Readme was updated accordingly